src/common/string/filter/sequence-matching.ts

```diff
@@ -63,9 +65,7 @@ type FuzzyFilterSort = <T extends ScorableTextItem>(
 export const fuzzyFilterSort: FuzzyFilterSort = (filter, items) => {
   return items
     .map((item) => {
-      item.score = item.altText
-        ? fuzzySequentialMatch(filter, item.filterText, item.altText)
-        : fuzzySequentialMatch(filter, item.filterText);
+      item.score = fuzzySequentialMatch(filter, item);
```

Comment:
Could just do:
```suggestion
      item.score = fuzzySequentialMatch(filter, item.words);
```

Reply:
Unfortunately not. The point here is that we need `fuzzySequentialMatch` to have access to the whole `item` so that we can efficiently calculate and store the "decorated text" (for letter highlighting) on the item itself during the fuzzy checking loop.

This is to avoid having to loop over all the matches after filtering to get the text decoration.